media-sound/forked-daapd/forked-daapd-23.2.ebuild

```diff
@@ -70,7 +73,7 @@ src_install() {
 
 	# dodir by itself fails in the likely case of /srv/music having a
 	# volume mounted already.
-	test -d /srv/music || dodir /srv/music
+	[ -d /srv/music ] || dodir /srv/music
```

Comment:
Sorry, the reversion was an artifact of my somehow not having it in my branch originally.  I've pushed the test change in another (non-squashed) commit.